tools/benchmark/src/MochaCustomOutputReporter.ts

```diff
@@ -0,0 +1,190 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// This file is a reporter used with node, so depending on node is fine.
+/* eslint-disable import/no-nodejs-modules */
```

Comment:
For what it's worth, I believe this entire tool is intended to be run in Node, so we could probably disable these rules for the entire package in its `.eslintrc` file.

Reply:
Right right. Updated the `.eslintrc` rule